OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -98,6 +100,7 @@ void CaptureClient::Capture(int32_t process_id, std::string process_name,
     ERROR("Sending CaptureRequest on Capture's gRPC stream");
     reader_writer_->WritesDone();
     FinishCapture();
+    capture_listener_->OnCaptureFailed(ErrorMessage("Error sending capture request."));
```

Comment:
FinishCapture can aslo generate failure.

Reply:
Should be not the case anymore, I i remove the error in FinishCapture as in https://github.com/google/orbit/pull/1205#discussion_r488711100